src/equations/maxwell_1d.jl

```diff
@@ -0,0 +1,104 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    MaxwellEquations1D
```

Comment:
```suggestion
    MaxwellEquations1D(c = 299_792_458.0)
```

to document the full constructor.